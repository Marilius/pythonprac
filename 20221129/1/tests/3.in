class C(metaclass=dump):
    def __init__(self, a, **b):
        self.a = self.f69(a)

    @staticmethod
    def f69(x, *a):
        return '69'

    def add(self, *a):
        self.a += self.f69(a)
        return self.a


c = C(10)
print(c.f69('золото'))
print(c.f69('курячий'))