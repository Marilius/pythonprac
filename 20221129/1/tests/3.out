__init__: (10,), {}
f69: (10,), {}
f69: ('золото',), {}
69
f69: ('курячий',), {}
69